src/pages/Homepage.tsx

```diff
@@ -50,14 +49,6 @@ const paperStyles = {
   width: '100%',
 };
 
-const avatarStyles = {
```

Comment:
A bunch of stuff was removed from Homepage.tsx, is this 1. Intentional, 2. Because they didn't do anything?